examples/example6.py

```diff
@@ -0,0 +1,102 @@
+import ctypes
+import os
+import sys
+import shutil
+
+
+# load vocaltractlab binary
+_FILE_ENDING = ''
+if sys.platform.startswith('linux'):
+    _FILE_ENDING = '.so'
+elif sys.platform.startswith('win32'):
+    _FILE_ENDING = '.dll'
+elif sys.platform.startswith('darwin'):
+    _FILE_ENDING = '.dylib'
+
+VTL = ctypes.cdll.LoadLibrary('../lib/libVocalTractLabApi' + _FILE_ENDING)
+del _FILE_ENDING
+
+
+# get version / compile date
+version = ctypes.c_char_p(b' ' * 64)
+VTL.vtlGetVersion(version)
+print('Compile date of the library: "%s"' % version.value.decode())
+
+
+# initialize vtl
+speaker_file_name = ctypes.c_char_p('../resources/JD3.speaker'.encode())
+
+failure = VTL.vtlInitialize(speaker_file_name)
+if failure != 0:
+    raise ValueError('Error in vtlInitialize! Errorcode: %i' % failure)
+
+
+# NOTE: You need the apfelsine.ges file, which will be create by running
+# example5.py
+
+print("Part 1: Synthesize gestural score as reference")
+speaker_file_name = ctypes.c_char_p('../resources/JD3.speaker'.encode())
+gesture_file_name = ctypes.c_char_p('apfelsine.ges'.encode())
+wav_file_name = ctypes.c_char_p('apfelsine.wav'.encode())
+feedback_file_name = ctypes.c_char_p('apfelsine.txt'.encode())
+
+VTL.vtlInitialize(speaker_file_name)
+failure = VTL.vtlGesturalScoreToAudio(gesture_file_name,  # input
+                          wav_file_name,  # output can be empty string "", if
+                                          # you don't want to save audio to file
+                          None,  # *audio
+                          None,  # *numSamples
+                          1)  # enableConsoleOutput
+if failure != 0:
+    raise ValueError('Error in vtlGesturalScoreToAudio! Errorcode: %i' % failure)
+    #print('Error in vtlGesturalScoreToAudio! Errorcode: %i' % failure)
+VTL.vtlClose()
+
+
+print("Part 2: export ema points only")
```

Comment:
Merge current main into this branch so that the EMA-related functions are available.